### ISSUE TITLE
forever stopbypid does not work

### ISSUE THREAD

**Issue by toddself** [foreverjs/forever#646]
Forever version 0.13.0

Here's trying to use `stopbypid`

```
root@35747a7d2ba4:/home/user/scripto2# forever list
info:    No forever processes running
root@35747a7d2ba4:/home/user/scripto2# forever start -a -l forever.log -o /var/log/server.js.log -e /var/log/server.js.err.log --pidFile /run/scripto.pid -c node scripto/ep_rundown/node/server.js
warn:    --minUptime not set. Defaulting to: 1000ms
warn:    --spinSleepTime not set. Your script will exit if it does not stay up for at least 1000ms
info:    Forever processing file: scripto/ep_rundown/node/server.js
root@35747a7d2ba4:/home/user/scripto2# forever stopbypid $(cat /run/scripto.pid)
error:   Forever cannot find process with pid: 493
root@35747a7d2ba4:/home/user/scripto2# forever list
info:    Forever processes running
data:        uid  command script                            forever pid id logfile               uptime
data:    [0] 1cMD node    scripto/ep_rundown/node/server.js 487     493    /.forever/forever.log 0:0:0:12.665
root@35747a7d2ba4:/home/user/scripto2#
```

**Comment by zogot** [comment]
Reporting the issue is still here. With the rollback from 14, any chance of getting this fixed?

**Comment by Tjatse** [comment]
It's a confirmed bug which has been fixed via [68502f1](https://github.com/foreverjs/forever/commit/68502f1b604eb52bead2d831daf4dd5f7dec82e4), since [54194df](https://github.com/foreverjs/forever/commit/54194dffbd0288b177592743c83b29d3ece881c3#diff-6d05bb41e13be0623bffa0826693104fR258). after v 0.13.0
